plugins/precompile-markdown/index.js

```diff
@@ -121,10 +103,12 @@ marked.use({
 
 			Prism.languages[lang] == null
 				? console.warn(`No Prism highlighter for language: ${lang}`)
-				: text = Prism.highlight(code, Prism.languages[lang], lang);
+				: (text = Prism.highlight(code, Prism.languages[lang], lang));
```

Comment:
Also apologies for the diff in this file, evidently I disabled my precommit hook at some point and forgot to re-enable it 😬